.gitignore

```diff
@@ -23,6 +23,7 @@ travis-deploy-key
 
 # distribution
 **/dist
+**/built
```

Comment:
/built is for the demo only?

Reply:
yep, the only `built` folder we have is in the demo package